tests/shell/Makefile

```diff
@@ -12,4 +12,10 @@ BOARD_BLACKLIST += chronos
 
 TEST_ON_CI_WHITELIST += all
 
+# In order to properly test and debug the shell it is better to use a terminal
```

Comment:
Why just set this in `tests/shell` only ? I would prefer to have this set globally in `Makefile.tests_common`.